### PR TITLE
[4.0] searchtools should also clear search field

### PR DESCRIPTION
**Pull request by mvanvu**
### Before PR
Click on the clear button to reset all input value but the input search is not cleared

### After PR
The search field should be cleared

### CODE REVIEW

File: media/system/js/searchtools.js [hunk 1]
```diff
@@ -195,7 +195,11 @@
 		},
 		clear: function () {
 			var self = this;
-
+			
+			if(self.searchField){
```
Comment (C-Lodder) [inline media/system/js/searchtools.js]: Please add spaces like so `if (self.searchField) {`

**Comment by C-Lodder** [comment]
You've also need to commit the `.min.js` file too

**Comment by ghost** [comment]
@mvanvu is this PR for an allready open Issue?

**Comment by mvanvu** [comment]
@franz-wohlkoenig No, it isn't

**Comment by ghost** [comment]
I have tested this item :red_circle: unsuccessfully on e37031e864d06f0d3c7c1c13b850bab8f31ca723

With and -out PR Searchfield isn't cleared after Click on "Clear"-Button. No Javascript-Errors in Web Developer-Console.

### System information
4.0.0-dev
Multilanguage Site (4 Lang.) & Sample Data
macOS Sierra, 10.12.6
Firefox 57 (64-bit)
#### MAMP 4.2
- PHP 7.0.22
- MySQLi 5.6.35
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18810">issues.joomla.org/tracker/joomla-cms/18810</a>.</sub>

**Comment by brianteeman** [comment]
@franz-wohlkoenig did you have debug enabled? if not then joomla will use the existing minified file which is not in the pr as mentioned above by @C-Lodder 

Also this looks to be a PR to fix #17195

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on e37031e864d06f0d3c7c1c13b850bab8f31ca723<br /><br />Tested in debug mode so that the non minified js could be used and the test is successful

please add the minified version to this PR
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18810">issues.joomla.org/tracker/joomla-cms/18810</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on e37031e864d06f0d3c7c1c13b850bab8f31ca723

@brianteeman thanks for Hint for "debug enabled".
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18810">issues.joomla.org/tracker/joomla-cms/18810</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Comment by C-Lodder** [comment]
@franz-wohlkoenig - The minified JS file is still missing. Please remove RTC till it has been committed

**Comment by ghost** [comment]
Status set on "Pending".<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18810">issues.joomla.org/tracker/joomla-cms/18810</a>.</sub>

**Comment by C-Lodder** [comment]
@mvanvu - Please minify the file properly using grunt or https://skalman.github.io/UglifyJS-online/

**Comment by wilsonge** [comment]
Thanks :)

**Pull request merged**